WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/FeatureAnnouncementCardView.swift

```diff
@@ -23,25 +23,31 @@ struct FeatureAnnouncementCardView: View {
                 Spacer()
                 if let dismiss = dismiss {
                     Button(action: {
-                        showingDismissAlert = true
+                        showingDismissActionSheet = true
                     }) {
                         Image(systemName: "xmark")
                             .foregroundColor(Color(.withColorStudio(.gray)))
                     }
                     .padding(.trailing, Layout.padding)
-                    .alert(isPresented: $showingDismissAlert,
-                           content: {
-                        Alert(title: Text(viewModel.dismissAlertTitle),
-                              message: Text(viewModel.dismissAlertMessage),
-                              primaryButton: .cancel(Text(Localization.remindLaterButton), action: {
-                            viewModel.remindLaterTapped()
-                            dismiss()
-                        }),
-                              secondaryButton: .default(Text(Localization.dontShowAgainButton), action: {
-                            viewModel.dontShowAgainTapped()
-                            dismiss()
-                        }))
-                    })
+                    .actionSheet(isPresented: $showingDismissActionSheet) {
+                        ActionSheet(
+                            title: Text(viewModel.dismissAlertTitle),
+                            message: Text(viewModel.dismissAlertMessage),
+                            buttons: [
+                                .default(Text(Localization.remindLaterButton),
+                                         action: {
+                                             viewModel.remindLaterTapped()
+                                             dismiss()
+                                         }),
+                                .default(Text(Localization.dontShowAgainButton),
```

Comment:
I'm seeing an odd behavior with localization on these Texts and button titles, I'm not sure if is my local config thought, can you replicate it? If you change the device language, some of these appear localized while others don't:

<img width=250 src="https://user-images.githubusercontent.com/3812076/180824837-977b831a-63bd-411b-ba24-36c72986c3c3.png">

While `cancel()` shows localized, as is a system button, the rest don't seem to be consistent. That said, possibly this is something to investigate on a different issue 👍

Reply:
Good point @iamgabrielma :) My assumption is that those two first strings are not localized yet, the translator team will do that before release. For the third, the Cancel button, we use the system cancel button that already translates the string automatically, without the need to translate it ourselves. Hence the behavior that you see, some aren't translated, while cancel is. :)

Reply:
> those two first strings are not localized yet, the translator team will do that before release.

Ah, gotcha! That makes sense! 👯